src/docs/cookbook/animation/page-route-animation.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Animate a page route transition
+next:
+  title: Animate the properties of a container
+  path: /docs/cookbook/animation/animated-container
+---
+
+# Animate a page route transition
+
+A design language, such as Material, defines standard behaviors when
+transitioning between routes (or screens). Sometimes, though, a custom
+transition between screens can make an app more unique. To help,
+[PageRouteBuilder](https://api.flutter.dev/flutter/widgets/PageRouteBuilder-class.html)
+provides an
+[Animation](https://api.flutter.dev/flutter/animation/Animation-class.html)
+object. This `Animation` can be used with
+[Tween](https://api.flutter.dev/flutter/animation/Tween-class.html) and
+[Curve](https://api.flutter.dev/flutter/animation/Curve-class.html) objects to
+customize the transition animation. This recipe shows how to transition between
+routes by animating the new route into view from the bottom of the screen.
+
+To create a custom page route transition, this recipe uses the following steps:
+
+1. Set up a PageRouteBuilder
+2. Create a `Tween`
+3. Add an `AnimatedWidget`
+4. Use a `CurveTween`
+5. Combine the two `Tween`s
+
+# 1. Set up a PageRouteBuilder
+
+To start, use a
+[PageRouteBuilder](https://api.flutter.dev/flutter/widgets/PageRouteBuilder-class.html)
+to create a [Route](https://api.flutter.dev/flutter/widgets/Route-class.html).
+`PageRouteBuilder` has two callbacks, one to build the content of the route
+(`pageBuilder`), and one to build the route's transition (`transitionsBuilder`).
+
+{{site.alert.note}}
+
+Note: The `child` parameter in transitionsBuilder is the widget returned from
```

Comment:
Delete the empty line above and below the note text. We indent notes by two chars, so:

{{site.alert.note}}
  The `child` parameter …
{{site.alert.end}}

Also, the site.alert macro inserts the “Note:” text, so you can delete that.

Reply:
👍 fixed